charts/argo-cd/templates/argocd-notifications/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.createClusterRoles }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "argo-cd.notifications.fullname" . }}
+  labels:
+    {{- include "argo-cd.labels" (dict "context" . "component" .Values.notifications.name "name" .Values.notifications.name) | nindent 4 }}
+rules:
+  {{- with .Values.notifications.clusterRoleRules.rules }}
+    {{- toYaml . | nindent 2 }}
+  {{- else }}
```

Comment:
If the following permission (line 12-21) is the minimum necessary one, then I think we can change  `else` to `end` ? 👀

Reply:
Done